Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -78,7 +78,7 @@
   "loc.messages.UnableToFetchContinuousWebJobs": "Unable to fetch continuous web jobs",
   "loc.messages.UnableToStartContinuousWebJob": "Unable to start continuous web job",
   "loc.messages.UnableToStopContinuousWebJob": "Unable to stop continuous web job",
-  "loc.messages.CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for azure. Status code: %s, status message: %s",
+  "loc.messages.CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for Azure. Status code: '%s', status message: %s",
```

Comment:
Do we need to wrap the status messages also in quotes?

Reply:
No there is no need of it . So removing it.